packages/manager/src/lib/requireResolve.ts

```diff
@@ -0,0 +1,30 @@
+import _module, { createRequire } from "node:module";
+
+/**
+ * Resolves a module path with `createRequire().resolve()` with Yarn PnP
+ * support.
+ *
+ * @param id - Module to resolve.
+ * @param from - Location to resolve the module from.
+ *
+ * @returns - Resolved module path.
+ */
+export default (id: string, from: string): string => {
+	let _id = id;
+
+	// Support Yarn PnP
+	if (
+		process.versions.pnp &&
+		"findPnpApi" in _module &&
+		_module.findPnpApi instanceof Function
```

Comment:
💡 #idea: `typeof _module.findPnpApi === "function"` seems more idiomatic to me. What do you think? Is there a benefit to using `instanceof`?

Reply:
That also works, not sure why I went `instanceof` here